UI/Selenium/Steps/HearingScheduleSteps.cs

```diff
@@ -46,7 +58,7 @@ private void EnterHearingSchedule(HearingSchedule hearingSchedule)
         public Hearing CreateHearingModel(Table table, int min = 3)
         {
             var tableRow = table.Rows[0];
-            var date = DateTime.Now.AddMinutes(min);
+            var date = DateTime.Now.AddMinutes(min).AddDays(2);
```

Comment:
Please remove, by default a hearing should be today unless there's a need to set a future date.

Reply:
this fix is required for test to run in firefox else test fails

Reply:
Checked the branch code to verify this has been removed.